string result

string IS_IN_HOME_POSE = in_home_pose
string IS_IN_IDLE_MODE = in_idle_mode
string DEMO_EXECUTED = demo_executed
string DEMO1_EXECUTED = demo1_executed
string DEMO2_EXECUTED = demo2_executed
string DEMO_TERMINATED = demo_terminated
string IS_ABORTED = aborted
string IS_PREEMPTED = preempted